pkg/ipmi/layer_types.go

```diff
@@ -224,4 +224,20 @@ var (
 			}),
 		},
 	)
+
```

Comment:
Nit: please remove the blank line. Eventually it might be an idea to group corresponding requests and responses, but for the time being let's keep it consistent.